R/derive_params_growth_height.R

```diff
@@ -37,6 +41,12 @@
 #'   Datasets `who_wt_for_lgth_boys`/`who_wt_for_lgth_girls` are for subject age < 730.5 days.
 #'   Datasets `who_wt_for_ht_boys`/`who_wt_for_ht_girls` are for subjects age >= 730.5 days.
 #'
+#'   If the `height` value from `dataset` falls between two `HEIGHT_LENGTH` values in
+#'   `meta_criteria`, then the `L`/`M`/`S` values that are chosen/mapped will be the
+#'   `HEIGHT_LENGTH` that has the smaller absolute difference to the value in `height`.
+#'   e.g. If dataset has a current age of 27.49 months, and the metadata contains records
+#'   for 27 and 28 months, the `L`/`M`/`S` corresponding to the 27 months record will be used.
```

Comment:
Replace the example with something relevant to height, instead of 27.49 months which is relevant to age still as copied from the other function